plugins/content/highlight/src/Component/Input/index.js

```diff
@@ -0,0 +1,58 @@
+import React, { Component } from "react"
+
+class Input extends Component {
+    render() {
+        const style = {
+            textarea: {
+                height: '250px',
+                width: '100%',
+                padding: '5px',
+                fontFamily: 'Menlo, Monaco, "Courier New", monospace'
+            },
+            submit: {
```

Comment:
I guess this styling isn't needed anymore, since the button was removed?

Reply:
totally right!

Reply:
will remove that